libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -164,7 +164,14 @@ const AP_Param::GroupInfo AP_Notify::var_info[] = {
     // @Range: 0 100
     // @Units: %
     AP_GROUPINFO("BUZZ_VOLUME", 8, AP_Notify, _buzzer_volume, 100),
-    
+
+    // @Param: NEOPIXEL_LENGTH
+    // @DisplayName: NeoPixel String Length
+    // @Description: The number of NeoPixel LEDs to use for notifications
+    // @Range: 1 10
```

Comment:
I think we can do up to 32

Reply:
I haven't been able to test this but I updated the range to 32